includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -388,6 +414,29 @@ private function update_is_test_mode_enabled( WP_REST_Request $request ) {
 		$this->gateway->update_option( 'testmode', $is_test_mode_enabled ? 'yes' : 'no' );
 	}
 
+	/**
+	 * Get the merchant payment method configuration in Stripe.
+	 *
+	 * @return array|null
+	 */
+	private function get_merchant_payment_method_configuration() {
+		$payment_method_configurations = $this->stripe_api->get_payment_method_configurations() && property_exists( $this->stripe_api->get_payment_method_configurations(), 'data' )
+			? $this->stripe_api->get_payment_method_configurations()->data
+			: null;
```

Comment:
What do we think about storing `$this->stripe_api->get_payment_method_configurations()` in an intermediate variable and doing the checks on that, to avoid multiple hits on the endpoint?

Something like
```suggestion
$result = $this->stripe_api->get_payment_method_configurations();
$payment_method_configurations = $result->data ?? null;
```

Reply:
Thank you @annemirasol! Updated [ebb6e90](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4091/commits/ebb6e90885011cb846191dd4f0a727935676bd9e)